activejob/test/cases/argument_serialization_test.rb

```diff
@@ -208,6 +209,8 @@ def assert_arguments_roundtrip(args)
     end
 
     def perform_round_trip(args)
-      ActiveJob::Arguments.deserialize(ActiveJob::Arguments.serialize(args))
+      ArgumentsRoundTripJob.perform_later(*args) # Actually performed inline
+
+      JobBuffer.last_value
```

Comment:
I made this change so that we're exercising the serialization by the adapter itself, not just Active Job's serialization.

Making this change without the addition of `BigDecimalSerializer` reveals the bug by causing the generated test for `BigDecimal` serialization earlier in this file to fail.